analyses/fusion_filtering/02-fusion-filtering.R

```diff
@@ -253,7 +281,7 @@ annotated_filtered_fusions<-annotate_fusion_calls(standardFusioncalls = expressi
 saveRDS(annotated_filtered_fusions,paste0(opt$outputfile,"_QC_expression_filtered_annotated.RDS"))
 
 # Annotate standardized fusion calls for project specific gene list filtering
-annotated_unfiltered_fusions<-annotate_fusion_calls(standardFusioncalls = standardFusioncalls,geneListReferenceDataTab = geneListReferenceDataTab ,fusionReferenceDataTab = fusionReferenceDataTab )
+annotated_unfiltered_fusions<-annotate_fusion_calls(standardFusioncalls = QCFiltered,geneListReferenceDataTab = geneListReferenceDataTab ,fusionReferenceDataTab = fusionReferenceDataTab )
```

Comment:
The variable name here is `annotated_unfiltered_fusions` - want to check that this is the intended object to be passed to `annotate_fusion_calls`?

Reply:
yeah I removed that bit, I was saving the unfiltered (dont pass QC and expression) but annotated by oncogenic genes if we wanted to check those those are not used in 03 and 04 so I can remove them from this code.